src/views/migrationpolicies/list/components/MigrationPolicyCreateForm/copmonents/BandwidthInput/BandwidthInput.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useCallback, useState } from 'react';
+
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { BinaryUnit, toIECUnit } from '@kubevirt-utils/utils/units';
+import {
+  NumberInput,
+  Select,
+  SelectOption,
+  SelectVariant,
+  Split,
+  SplitItem,
+} from '@patternfly/react-core';
+
+type BandwidthInputProps = {
+  bandwidth: {
+    value: number;
+    unit: BinaryUnit;
+  };
+  setBandwidth: React.Dispatch<
+    React.SetStateAction<{
+      value: number;
+      unit: BinaryUnit;
+    }>
+  >;
+};
+
+const unitOptions = [BinaryUnit.Ki, BinaryUnit.Mi, BinaryUnit.Gi];
+
+const BandwidthInput: React.FC<BandwidthInputProps> = ({ bandwidth, setBandwidth }) => {
+  const { t } = useKubevirtTranslation();
+  const [isQuantitySelectOpen, setIsQuantitySelectOpen] = useState<boolean>(false);
+
+  const onSelectUnit = useCallback(
+    (
+      event: React.MouseEvent<Element, MouseEvent> | React.ChangeEvent<Element>,
+      newUnit: BinaryUnit,
+    ) => {
+      setBandwidth((prev) => ({ ...prev, unit: newUnit }));
+      setIsQuantitySelectOpen(false);
+    },
+    [setBandwidth],
+  );
+
+  return (
+    <Split hasGutter>
+      <SplitItem>
+        <NumberInput
+          min={0}
+          value={bandwidth?.value}
+          onMinus={() => setBandwidth((prev) => ({ ...prev, value: --prev.value }))}
+          onChange={(event) =>
+            +event.currentTarget.value >= 0 &&
```

Comment:
Why `+` at the beginning?

Reply:
the value which we get is of string type, the `+` is a shortend conversion string to number, so I can make this kind of evaluation :)

Reply:
Cool